hyperjumptech/monika
# fix: error sent as a body to Symon

# Monika Pull Request (PR)

## What feature/issue does this PR add

1. This PR fixes `error` sent as a` body` to Symon
2. This PR renames the `errMessage` to `error`

## How to test

1. Run Monika for both Symon and non-Symon mode
3. Make sure the probing scenario is same as current implementation (no error, not changed)
